src/components/hooks/__mocks__/useMeeting.js

```diff
@@ -1,11 +1,14 @@
 import {useContext} from 'react';
 
-export default function useMeeting(meetingID) {
+export default function useMeeting(meetingID, meetingDestination) {
   const datasource = useContext();
+  let result = {};
 
-  if (!(meetingID in datasource.meetingsAdapter)) {
-    throw new Error(`Could not find meeting with ID "${meetingID}"`);
+  if (meetingDestination) {
+    result = datasource.meetingsAdapter.localMedia;
+  } else if (meetingID in datasource.meetingsAdapter) {
```

Comment:
Where are you handling the errors here?

Reply:
This is a mock, i don't think we have any test cases where we need to have the error thrown

Reply:
sure

Reply:
all the mocks are pretty inconsistent. so whatever.

Reply:
Having an error also makes one of the loading stories fail